google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -111,25 +104,20 @@ def __post_init__(self, document_text) -> None:
 
 
 @dataclasses.dataclass
-class Token:
-    """Represents a wrapped documentai.Document.Page.
-    .
-
-    Attributes:
-        documentai_object (google.cloud.documentai.Document.Page.Token):
-            Required. The original google.cloud.documentai.Document.Page.Token object.
-        _page (Page):
-                    Required. The Page object.
-    """
+class PageElement:
+    """Base class for representing a wrapped Document AI Page element (Symbol, Token, Line, Paragraph, Block)."""
```

Comment:
If this class is meant to be used only as a base class and not instantiated by the library, maybe make it an abstract class.  (Maybe also consider renaming it to something like "BasePageElement" or "PageElementBase" to ensure that future developers of the library can easily recognize its role.)

Reply:
Thanks for the suggestion!